template/nuxt/nuxt.config.js

```diff
@@ -91,6 +87,22 @@ module.exports = {
     // See https://github.com/nuxt-community/axios-module#options
   },<% } %>
 
+  <% if (ui === 'vuetify') { %>
+    /*
+    ** vuetify module configuration
```

Comment:
Can you please add a link to the docs here? https://github.com/nuxt-community/vuetify-module

Reply:
Thank you!
I added this link.